test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignerTests.cs

```diff
@@ -88,40 +89,38 @@ public SignerTests(SigningTestFixture fixture)
             using (var test = new Test(_testFixture.TrustedTestCertificateNotYetValid.Source.Cert))
             {
                 var exception = await Assert.ThrowsAsync<SignatureException>(
-                    () => test.Signer.SignAsync(test.Request, NullLogger.Instance, CancellationToken.None));
+                    () => test.Signer.SignAsync(CancellationToken.None));
 
                 Assert.Equal(NuGetLogCode.NU3017, exception.Code);
                 Assert.Contains("The signing certificate is not yet valid", exception.Message);
 
-                var isSigned = await IsSignedAsync(test.WriteStream);
-
+                var isSigned = await IsSignedAsync(test.Options.PackageFilePath);
                 Assert.False(isSigned);
+
+                Assert.False(File.Exists(test.Options.OutputFilePath));
+
             }
         }
 
-        private static async Task<bool> IsSignedAsync(MemoryStream package)
+        private static async Task<bool> IsSignedAsync(string packagePath)
         {
-            var currentPosition = package.Position;
-
-            var reader = new PackageArchiveReader(package, leaveStreamOpen: true);
-
-            var isSigned = await reader.IsSignedAsync(CancellationToken.None);
+            using (var package = File.OpenRead(packagePath))
+            {
+                var reader = new PackageArchiveReader(package, leaveStreamOpen: true);
```

Comment:
`reader` is also disposable.  :)  However, I can fix it in my upcoming PR where I dedupe `IsSignedAsync(...)` calls.